src/Castle.Facilities.AspNetCore/Castle.Facilities.AspNetCore.csproj

```diff
@@ -17,6 +17,10 @@
 		<RootNamespace>Castle.Facilities.AspNetCore</RootNamespace>
 	</PropertyGroup>
 
+	<ItemGroup>
+	  <None Remove="Resolvers\FrameworkConfigurationDependencyResolver.cs~RF4e009575.TMP" />
```

Comment:
Looks like some junk added to the project file.

Reply:
Good spot, will remove.

Reply:
Done